test.sh

```diff
@@ -21,6 +22,18 @@ INTEGRATION_FLAGS=()
 FILTER=()
 COVERAGE="false"
 COVERAGE_DIR="test/coverage/$(date +%Y-%m-%d_%H-%M-%S)"
+DB_URL_FILES=(
+  backfiller_dburl
+  badkeyrevoker_dburl
+  cert_checker_dburl
+  expiration_mailer_dburl
+  incidents_dburl
+  mailer_dburl
+  ocsp_responder_dburl
+  revoker_dburl
+  sa_dburl
+  sa_ro_dburl
```

Comment:
We can delete backfiller, expiration_mailer, mailer, and ocsp_responder from this list, from the gitignore, and from all the directories where these files live. They're not referenced by any configs.